twilio-sheet/send-sms/Code.js

```diff
@@ -0,0 +1,74 @@
+// Spreadsheet column names mapped to 0-based index numbers.
+var TIME_ENTERED = 0;
+var PHONE_NUMBER = 1;
+var MESSAGE = 2;
+var STATUS = 3;
+
+// Enter your Twilio account information here.
+var TWILIO_ACCOUNT_SID = 'PASTE_TWILIO_ACCOUNT_ID_HERE_EX: ACa0072e7863c5d13b703e1e18f7cab1ee';
+var TWILIO_SMS_NUMBER = 'PASTE_TWILIO_SMS_NUMBER_HERE_EX: 14155925860';
+var TWILIO_AUTH_TOKEN = 'PASTE_TWILIO_AUTH_TOKEN_HERE_EX: AC4z072e7863c5d13b703e1e18f7cab1ee:046077789202b8d7166de7a0d3383d9f';
+
+/**
+ * Installs a trigger in the Spreadsheet to run upon the Sheet being opened.
+ * To learn more about triggers read: https://developers.google.com/apps-script/guides/triggers
```

Comment:
It's usually a good idea to have the URL in its own line since URLs can be very large. It also makes it more evident that it's there.